README.md

```diff
@@ -0,0 +1,35 @@
+# CASS-Degrees-Code
+Primary Codebase for the TechLauncher CASS Degrees project
+
+For our landing page please go to: https://github.com/cass-degrees/CASS-Degrees
+
+## Our Git Branching Structure
+In this project we will have two main branches: **Master** and **Develop**
+
+The **Master** branch will only contain completed and working releases that have addressed all the user stories of the relevant release. 
+Merges to this branch will only come from the develop branch once a release in develop has been completed and finalised.
+
+The **Develop** branch is the main working branch which will have completed and working user stories for the relevant release that the team is working on.
+No one is to directly commit to this branch, it is updated by merges from branches created for the relevant user story or feature that a developer is working on.
+
+The idea is that once someone completes a feature or user story, they will create a new branch with their code (with an appropriate name) and request a *Pull Request* into
+the develop branch.
+
+**Pull Requests** are reviewed by the projects team members, the rule is that 2 people must approve of the feature or user story before it can
+be merged into the develop branch. Alternatively, if it has been seen by the project team members and has been open for more than 2 days, then the
```

Comment:
This isn't exactly correct - make sure to review the decision log for this

Reply:
Yeah I wasn't really sure about the changes, I'll update this bit